src/test/data/JsonAddressBookStorageTest/invalidAndValidPersonAddressBook.json

```diff
@@ -3,11 +3,13 @@
     "name": "Valid Person",
     "id": "P12345",
     "ward" : "A1",
-    "diagnosis" : "E. coli infection"
+    "diagnosis" : "E. coli infection",
+    "medication" : "Ciprofloxacin (Cipro)"
   }, {
-    "name": "Person With Invalid Id and Ward Field",
+    "name": "Person With Invalid Id, Ward and Medication Field",
```

Comment:
Is the medication field invalid? Also not too sure but I feel that this class was designed just for comparison of a valid and invalid person (i.e. actually one field invalid can already?)